users/models.py

```diff
@@ -311,3 +311,9 @@ def is_complete(self):
     def __str__(self):
         """Str representation for the profile"""
         return f"Profile for {self.user}"
+
+
+class BlockList(TimestampedModel):
+    """ A user's blocklist model """
+
+    hashed_email = models.CharField(max_length=128, blank=True, default="")
```

Comment:
I don't think we should be allowing a blank value here because there's no use case for that.